tasks/actorstate/miner.go

```diff
@@ -385,6 +385,16 @@ func ExtractMinerSectorData(ctx context.Context, ec *MinerStateExtractionContext
 				})
 			}
 		}
+		for _, snappedSector := range sectorChanges.Snapped {
+			for _, dealID := range snappedSector.To.DealIDs {
+				sectorDealsModel = append(sectorDealsModel, &minermodel.MinerSectorDeal{
+					Height:   int64(ec.CurrTs.Height()),
+					MinerID:  a.Address.String(),
+					SectorID: uint64(snappedSector.To.SectorNumber),
+					DealID:   uint64(dealID),
+				})
+			}
+		}
```

Comment:
I assume this means we're missing data in production? Am I correct in thinking we can just re-walk the epochs since the v15 upgrade?

Reply:
Sadly yes, and thankfully yes. --from=`1594679` --to=`current_height`